apps/web/screens/queries/UniversityGateway.ts

```diff
@@ -73,21 +73,6 @@ export const GET_UNIVERSITY_GATEWAY_PROGRAM = gql`
       costInformationEn
       costInformationIs
       costPerYear
-      courses {
```

Comment:
Dumb question but why are we removing this, is this no longer being returned by the backend? What does the UI now look like, are there no courses being displayed anymore or are they queried from somewhere else now?

Reply:
the courses and programs are different, courses are not being displayed at the moment on the frontend and will not be there for a while since the universities do not have services that feed us information about those, but the frontend will not change, even if I remove this